libs/libzauth/libzauth-c/src/lib.rs

```diff
@@ -177,7 +176,6 @@ pub extern fn zauth_token_delete(t: *mut ZauthToken) {
 }
 
 #[repr(C)]
-#[no_mangle]
 #[derive(Clone, Copy, Debug)]
```

Comment:
@comawill this is the place I'm least sure about my change: the no_mangle at the enum level led to warnings; but I'm not sure if removing them is okay to do here.

Reply:
I think it's fine.  I don't think enum names or enum value names are part of the C ABI.

Reply:
I think `no_mangle` on structs and enums doesn't do anything, so it is safe to remove.

Reply:
thanks for your reviews!